README.md

```diff
@@ -105,7 +105,10 @@ curl -s "http://web.archive.org/cdx/search/cdx?url=*.domain.com/*&output=text&fl
 ```bash
 curl -s "https://jldc.me/anubis/subdomains/domain.com" | grep -Po "((http|https):\/\/)?(([\w.-]*)\.([\w]*)\.([A-z]))\w+" | sort -u
 ```
-
+### Brute subs with gobuster.
```

Comment:
This seems like a standard command, anyone can do it by read the docs in associated tool.

Reply:
> This seems like a standard command, anyone can do it by read the docs in associated tool.

ohh looks like you are right. and maybe I haven't read the entire document. thanks bro